fe/app/components/Divider/Divider.tsx

```diff
@@ -0,0 +1,18 @@
+import { DividerProps } from "@/app/components/Divider/Divider.models";
+import clsx from "clsx";
+import React from "react";
+
+const Divider = ({ category, orientation = "horizontal" }: DividerProps) => {
+  return orientation === "horizontal" ? (
+    <hr className={clsx("w-full", "border-" + category)} />
+  ) : (
+    <div
+      className={clsx(
+        "w-[1px] self-stretch",
+        category ? "bg-" + category : "bg-white"
```

Comment:
Just a suggestion, here´s another way of doing the className without clsx 
className=`w-[1px] self-stretch bg-${category ?? "white"}`

Reply:
thanks! updated